Note regarding image visibility for odo deploy (#6191)
redhat-developer/odo: docs/website/docs/user-guides/advanced/deploy/_running_deploy.mdx

Your application has now been deployed to the Kubernetes cluster with Deployment, Service, and Ingress resources.

:::note

> Depending upon the container registry which you specified in the `devfile.yaml`, the access for container image

Reviewer: ```suggestion
Depending upon the container registry specified in the `devfile.yaml`, the access for container image
```

> pushed by `odo deploy` might be private by default, e.g., images pushed to [Quay.io](https://quay.io). Change the

Reviewer: ```suggestion
pushed by `odo deploy` may be private by default. For example: images pushed to [Quay.io](https://quay.io). Change the
```

> image visibility appropriately to be able to run `odo deploy` successfully.

Reviewer: ```suggestion
image visibility appropriately in order to run `odo deploy` successfully.
```
